crypto/params_from_text.c

```diff
@@ -62,20 +63,25 @@ static int prepare_from_text(const OSSL_PARAM *paramdefs, const char *key,
          * buffer, i.e. if it's negative, we need to deal with it.  We do
          * it by subtracting 1 here and inverting the bytes in
          * construct_from_text() below.
+         * To subtract 1 from an absolute value of a negative number we
+         * actually have to add 1.
```

Comment:
Could you an example here?  Something like this: -3 - 1 = -4, |-3| = 3 + 1 = 4.

Reply:
I'm approving this without the example, so adding it is optional.

Nice work BTW.

Reply:
I just added the example as suggested.